source/include/core_http_client.h

```diff
@@ -672,6 +744,46 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * - #HTTP_SECURITY_ALERT_INVALID_STATUS_CODE
  * - #HTTP_SECURITY_ALERT_INVALID_CHARACTER
  * - #HTTP_SECURITY_ALERT_INVALID_CONTENT_LENGTH
+ *
+ * <b>Example</b>
+ * @code{c}
+ * // Variables used in this example.
+ * HTTPStatus_t httpLibStatus = HTTP_SUCCESS;
+ * TransportInterface_t transportInterface = { 0 };
+ * HTTPResponse_t = { 0 };
+ * char requestBody[] = "This is example request body.";
+ *
+ * // Assume that requestHeaders has been initialized with
+ * // HTTPClient_InitializeResponseHeaders() and any headers needed have been
```

Comment:
```suggestion
 * // HTTPClient_InitializeResponseHeaders() and any desired headers have been
```

Needed seems to indicate that the call to `HTTPClient_AddHeader` is required for the request.

Reply:
Sure ill write "additional"